docs/content/technologien.tex

```diff
@@ -1,7 +1,136 @@
+
+Abbildung \ref{fig:store-definition} zeigt einen Vergleich desselben Codes in CSS und Sass. Durch Sass kann hierbei redundanter Code verhindert werden. Für Interessierte sei für weitere Funktionen auf \cite{SassGuide} verwiesen.
+
+\begin{figure}[!htb]
+  \begin{minipage}{0.48\textwidth}
+    \centering
+    \includegraphics[width=.95\linewidth]{images/css-example.png}
+  \end{minipage}\hfill
+  \begin{minipage}{0.48\textwidth}
+    \centering
+    \includegraphics[width=1\linewidth]{images/sass-example.png}
+  \end{minipage}
+  \caption[Vergleich von CSS und Sass]{Vergleich von CSS (links) und Sass (rechts) am Beispiel von Verschachtlung \cite{SassGuide}}
+  \label{fig:store-definition}
+\end{figure}
+
+%
+% Bootstrap
+%
+\section{Bootstrap}
+Um die Gestaltung der Benutzeroberfläche (siehe Kapitel \ref{sec:struktur-frontend}) zu vereinfachen, wird Bootstrap verwendet. Bootstrap ist Werkzeugsatz, der u.A. vorgefertigte Komponenten bzw. CSS sowie Icons bereitstellt \cite[vgl.][]{Bootstrap}.
+
+%
+% Vue
+%
+\section{Vue}
+
+Für die Erstellung der Benutzeroberfläche (im Folgenden \glqq Frontend\grqq{} genannt) wird Vue (Version 3.2) verwendet. Vue ist ein progressives JavaScript-Framework, das die Frontend-Entwicklung vereinfachen soll. Progressiv bedeutet hierbei, das es für das gesamte Frontend oder auch nur für Teile dessen verwendet werden kann. Vue kann folglich mit anderen Technologien verwendet oder Schritt-für-Schritt (progressiv) in bereits vorhandene Projekte integriert werden \cite[vgl.][]{VueIntroduction}.
+
+%
+% Vue - Grundlagen von Komponenten
+%
+\subsection{Grundlagen von Komponenten}
+\begin{figure}[H]
+  \includegraphics[width=0.9\textwidth]{images/vue-components.png}
+  \centering
+  \caption[Vue-Komponenten]{Strukturierung des Frontends durch Vue-Komponenten \cite{VueComponentBasics}}
+  \label{fig:vue-components}
+\end{figure}
+
+Vue-Komponenten erlauben die Unterteilung des Frontends in unabhängige, isolierte und wiederverwendbare Teile. So wird das Frontend in kleinere Komponenten gegliedert (siehe Abbildung \ref{fig:vue-components}), die dann die benötigte Logik kapseln und unabhängig entwickelt werden können \cite[vgl.][]{VueComponentBasics}.
+
+Vue bietet mehrere Möglichkeiten, um eine Komponente zu definieren. Im Rahmen dieser Studienarbeit werden sogenannte \aclu{SFC} (\acs{SFC}) verwendet (siehe Abbildung \ref{fig:vue-example}). Jede Komponente wird hierbei in einer Datei mit der Dateiendung \lstinline{.vue} definiert. Nachdem die Komponente in einer anderen Datei importiert wurde, kann sie wie ein herkömmliches HTML-Element verwendet werden (siehe Abbildung \ref{fig:vue-example-usage}) \cite[vgl.][]{VueComponentBasics}.
+
+\begin{figure}[!htb]
+  \includegraphics[width=0.9\textwidth]{images/vue-example.png}
```

Comment:
convert png into eps files using inkscape

Reply:
Latex cannot handle .eps files, they will be converted to pdf on compilation. Also what is the benefit of converting it?